snapshot-labs/sx-ui
# feat: replace Twitter with X

### Summary

Closes: https://github.com/snapshot-labs/sx-ui/issues/784

<img width="453" alt="image" src="https://github.com/snapshot-labs/sx-ui/assets/1968722/4ae784eb-2096-4467-baa7-592f31713ee5">
<img width="497" alt="image" src="https://github.com/snapshot-labs/sx-ui/assets/1968722/17b902ed-a799-41fc-b45b-c2d1577eee04">


### How to test

Test with EVM networks until [this PR](https://github.com/snapshot-labs/sx-api/pull/196) is merged.

1. Check http://localhost:8080/#/sep:0xff7c7b0763a192a65b2a313880c4955403f257ec
2. Try editing space (it should show `X (Twitter)`.
